TACC/Core-Portal
# Task/fp 884 modal title styles

## Overview: ##

- Fix font size of modal header title.
- Consistent/Global styles for modal header.

> __Note__: The `x`/`X`/`×` close button character discrepancy would be solved in [FP-223](https://jira.tacc.utexas.edu/browse/FP-223)

## Related Jira tickets: ##

* [FP-884](https://jira.tacc.utexas.edu/browse/FP-884)

## Summary of Changes: ##

- __New__: Remove modal header/title styles from Allocations, Data Files, and Ticket modals.

## Testing Steps: ##
For each of the following modals, confirm the header has:
- background color `#F4F4F4`
- padding: vertical ~21.6px, horizontal ~28px
- color `#484848`
- font: Roboto, weight 400, size ~19.2px
- long text truncated with an ellipsis\*

> \* You can test this by editing any text in the live HTML of the modal title so that it is long enough to overflow the modal header.

Modals to check:
- Manage Allocations
- Data Files (any)
- Manage Account, "Edit […] Information"
- Ticket Create
- Ticket (View)
- Job History (View Details)

For the modals listed above, and any or all that were not, verify that:
- modal content, header, body, and footer\* do **not** have a border radius
- modal content, header, body do **not** have border
- modal footer **still** has a top border

> \* All or most of the Data Files modals have a footer you can test.

## UI Photos:

<details><summary>Click to show all</summary>

Manage Account Change Password\
![Manage Account Change Password](https://user-images.githubusercontent.com/62723358/106536860-d8df2a80-64be-11eb-9d54-e67c3481bbdd.png)

Manage Account Edit … Information\
![Manage Account Edit … Information](https://user-images.githubusercontent.com/62723358/106536864-daa8ee00-64be-11eb-8220-8b02015b6136.png)
![Job History View Details](https://user-images.githubusercontent.com/62723358/106536866-db418480-64be-11eb-81e2-6a0f19daab72.png)

Data Files Copy\
![Data Files Copy](https://user-images.githubusercontent.com/62723358/106536867-dbda1b00-64be-11eb-9366-0b196cc86d88.png)

Allocations Manage\
![Allocations Manage](https://user-images.githubusercontent.com/62723358/106536868-dbda1b00-64be-11eb-9fb2-423d198f6b14.png)

Ticket Create (post #331 merge)\
![Ticket Create (post #331 merge)](https://user-images.githubusercontent.com/62723358/106675769-58323400-657b-11eb-8d9c-93e5cb682423.jpeg)

<details><summary>Ticket Create (pre #331 merge)</summary>

![Ticket Create (pre #331 merge)](https://user-images.githubusercontent.com/62723358/106536870-dbda1b00-64be-11eb-89f2-5f75017709c2.png)\
The `.modal-footer` border is not showing here because at the time of this screenshot, the submit button was not inside a `.modal-footer` (that change happened after #331 was merged).

</details>

Ticket View\
![Ticket View](https://user-images.githubusercontent.com/62723358/106536871-dc72b180-64be-11eb-8d29-7cf2825d4943.png)

Allocations View Team\
![Allocations View Team](https://user-images.githubusercontent.com/62723358/106536872-dc72b180-64be-11eb-8592-154a1ba60f87.png)

Data Files Upload\
![Data Files Upload](https://user-images.githubusercontent.com/62723358/106536874-dc72b180-64be-11eb-92c3-3a705f81e29c.png)

</details>

## Notes: ##
